packages/grid/x-grid-modules/src/components/viewport.tsx

```diff
@@ -46,7 +46,7 @@ export const Viewport: ViewportType = React.forwardRef<HTMLDivElement, ViewportP
             rowIndex={renderCtx.firstRowIdx + idx}
             domIndex={idx}
           />
-          <RightEmptyCell key="right-empty" width={renderCtx.rightEmptyWidth} />
+          <RightEmptyCell width={renderCtx.rightEmptyWidth} />
```

Comment:
`key` has no impact, the position of the children is known by React, it never changes.